modules/network-modify-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+// * networking/configuring-network-operator.adoc
+
+[id="network-modifying-config-{context}"]
+= Modifying network configuration in a running cluster
+
+The Network Operator reconciles the state of the cluster against a desired
+configuration.
+
+You can modify only `kubeProxy` configuration parameters in a running cluster.
```

Comment:
You can also create new additionalNetworks

Reply:
What is `additionalNetworks`?

Reply:
@kalexand-rh, when you say separate assemblies, at what level of abstraction? Does this mean the `network-modify-config.adoc` file is split into two different files, one for each use case (`additionalNetworks` and `kubeProxy`), and then included in the original assembly?

Or does this mean creating second level assemblies and nesting each of these in `networking/configuring-network-operator.adoc`?

Or some other approach?

Thanks.

Reply:
None of these. If they're separate use cases that users are unlikely to need to address together, I think thatthey should be described in different assemblies in the networking folder.

Reply:
Thanks! I hadn't considered that approach.

Reply:
Sure! I think we had most of the discussions about how to map a single user goal/story to a single assembly before you joined the team. The "1 user goal = 1 assembly" idea is also what's driving the need to have a separate installation assembly to cover networking.

Reply:
@squeed, what are valid parameters for `additionalNetworks`?

Reply:
@jboxman they are any of the extra networks provided by Multus. The only ones supported in 4.1 are SRIOV and RawCNI.